evaluator/logger.go

```diff
@@ -83,7 +83,7 @@ func mapToArray(m map[string]interface{}) []interface{} {
 }
 
 func newLogger() logging.Logger {
-	lvl := zap.NewAtomicLevelAt(zapcore.InfoLevel)
+	lvl := zap.NewAtomicLevelAt(zapcore.DebugLevel)
```

Comment:
I guess I'm out of sync here. What's the reason we decided to setup this logger instead of inheriting from the cloudbeat/libbeat logger?